src/bench.h

```diff
@@ -9,41 +9,82 @@
 
```

Comment:
```suggestion
#include <stdint.h>
```

This currently builds fine because `util.h` is included in every `bench_` program. But that's somewhat fragile.

Reply:
thanks, added commit